index.html

```diff
@@ -0,0 +1,695 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="/assets/Favicon/favicon_io/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/assets/Favicon/favicon_io/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/assets/Favicon/favicon_io/favicon-16x16.png">
+    <link rel="manifest" href="/assets/Favicon/favicon_io/site.webmanifest">
+    <script src="https://kit.fontawesome.com/83847ae630.js" crossorigin="anonymous"></script>
+    <title>Tarang</title>
+</head>
+
+<body>
+    <!-- SECTION-1 -->
+    <header>
+        <nav class="navbar">
+            <h1><span class="ht-t">T</span><span class="ht-a">A</span><span class="ht-g">A</span><span
+                    class="ht-r">R</span><span class="ht-aa">A</span><span class="ht-n">N</span><span
+                    class="ht-g">G</span></h1>
+            <button class="toggle toggle-blue" onclick="toggle('toggle-list')">
```

Comment:
class toggle, toggle-blue, toggle-list need to be updated, as it doesn't describe the navigation inside the hamburger menu

Reply:
done